prisma/schema.prisma

```diff
@@ -170,12 +170,12 @@ model ProjectIdea {
 
 model ProjectIdeaVote {
   id            Int               @id @default(autoincrement())
-  votedBy       VoyageTeamMember? @relation(fields: [userId], references: [id], onUpdate: Cascade, onDelete: SetNull)
-  userId        Int?
-  ProjectIdea   ProjectIdea?      @relation(fields: [projectIdeaId], references: [id], onDelete: Cascade)
-  projectIdeaId Int?
+  votedBy       VoyageTeamMember? @relation(fields: [voyageTeamMemberId], references: [id], onUpdate: Cascade, onDelete: SetNull)
```

Comment:
I think for this onDelete should be Cascade -> if a voyageTeamMember is ever deleted (don't think it will happen, they will likely just have status "dropped"), we want to delete the vote as well.

SetNull on Project idea table is correct as we want to keep the idea (if they ever get moved or dropped)

we could just add this to the list of db updates

Reply:
I updated this a re-squashed the migrations